File: src/models/alert.py
from ..App import db

class Alert(db.Model):
    __tablename__ = 'alerts'
    
    id = db.Column(db.Integer, primary_key=True)
    vaccine_name = db.Column(db.String(100), nullable=False)
    due_date = db.Column(db.Date, nullable=False)
    alert_date = db.Column(db.Date, nullable=False)
    vaccine_id = db.Column(db.Integer, db.ForeignKey('vaccines.id'), nullable=False)
    pet_id = db.Column(db.Integer, db.ForeignKey('pets.id'), nullable=False)
    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)

File: src/Components/Alerts/Alert.js
import React from "react";
import DeleteAlertButton from "./DeleteAlertButton";

function Alert({ alert, onDelete }) {

    return (
        <div>
            <p>Vaccine: {alert.vaccine_name}</p>
            <p>Alert Date: {alert.alert_date}</p>
            <DeleteAlertButton onDelete={onDelete} />
        </div>
    )
}

export default Alert;